src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,15 +2,18 @@
 
 ## Unreleased
 
-* Fix issue where when an application has an ExceptionFilter, the exception data
-  wouldn't be collected.
-  ([#3475](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3475))
+* Updated DiagnosticSource event subscription to specific set of events.
```

Comment:
```suggestion
* AspNetCore instrumentation causes no extra allocation now as we have now updated DiagnosticSource event subscription to a specific set of events.
```

Reply:
> AspNetCore instrumentation causes no extra allocation

^ Ony if this is factually correct

Reply:
There is some extra allocation - it is not exactly 0. I will try with `params` to see if we can show the difference.